config/base/crd/bases/olm.operatorframework.io_clusterextensions.yaml

```diff
@@ -105,6 +105,9 @@ spec:
                 maxLength: 48
                 pattern: ^[a-z0-9]+(-[a-z0-9]+)*$
                 type: string
+                x-kubernetes-validations:
+                  - message: packageName is immutable
+                    rule: self == oldSelf
```

Comment:
Looks like there might be some formatting issues here causing the sanity/verify check to fail.

Reply:
seems to have passed the sanity check, but still fails on tilt.

Reply:
I think the tilt check is currently expected to fail based on some recent changes that were made in catalogd. I don't think it is a required check.